README.md

```diff
@@ -14,7 +14,9 @@ You can get your invite [here](http://slack.k8s.io/)
 ## Quick Start
 
 To deploy the cluster you can use :
-
+### If you are using Ubuntu versions, Please enable system-resolve to avoid error when init first master:
+#systemctl start system-resolve
+#systemctl enable system-resolve
```

Comment:
This seem invalid against markdown style.
Please check mrkdownlint job as https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/2008520837
```
$ markdownlint $(find . -name '*.md' | grep -vF './.git') --ignore docs/_sidebar.md --ignore contrib/dind/README.md
./README.md:17 MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Above] [Context: "### If you are using Ubuntu versions, Please enable system-resolve to avoid error when init first master:"]
./README.md:17:105 MD026/no-trailing-punctuation Trailing punctuation in heading [Punctuation: ':']
./README.md:18:1 MD018/no-missing-space-atx No space after hash on atx style heading [Context: "#systemctl start system-resolv..."]
./README.md:19:1 MD018/no-missing-space-atx No space after hash on atx style heading [Context: "#systemctl enable system-resol..."]
./README.md:20 MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Above] [Context: "### Ansible"]
```